win-create-projects.bat

```diff
@@ -9,6 +9,13 @@ utils\premake5.exe install_unifont
 rem Generate solutions
 utils\premake5.exe vs2022
 
+rem Check utils folder
+echo ------------------
+dir /B .\utils\
+echo ------------------
+dir /B .\utils\DXFiles
+echo ------------------
+
```

Comment:
This looks like debug code. It's not needed.
```suggestion
```

Reply:
> This looks like debug code. It's not needed.

Yeah, I tried to pinpoint the culprit. Couldnt do it without commiting ;/